src/SourceBuild/content/Directory.Build.props

```
@@ -189,4 +189,8 @@
     <PoisonUsageReportFile>$(PackageReportDir)poison-usage.xml</PoisonUsageReportFile>
   </PropertyGroup>
 
-</Project>
+  <PropertyGroup>
+    <VerticalName Condition="'$(VerticalName)' == ''">DefaultVertical</VerticalName>
+    <MergedAssetManifestOutputPath>$(ArtifactsDir)$(VerticalName).xml</MergedAssetManifestOutputPath>
+  </PropertyGroup>
+</Project>
```

style nit. We have an empty line after the project opening tag and before the project closing tag.

```suggestion

</Project>
```